clang/test/SemaCXX/member-template-specialization.cpp

```diff
@@ -0,0 +1,33 @@
+// RUN: %clang_cc1 -verify -fsyntax-only %s
```

Comment:
Normally we put lone tests for a specific bug report in its own file e.g. GH70735.cpp for this case or we find a test that covers a similar area and put the test in a namespace named after the issue e.g. GH70735.

Reply:
Can it be more elaborate like `GH70735-member-template-specialization.cpp`?

Reply:
gotcha, will do.